lib/SPIRV/libSPIRV/SPIRVEntry.h

```diff
@@ -310,6 +310,11 @@ class SPIRVEntry {
   std::vector<SPIRVWord>
   getMemberDecorationLiterals(Decoration Kind, SPIRVWord MemberNumber) const;
   std::vector<std::string> getDecorationStringLiteral(Decoration Kind) const;
```

Comment:
We can use getAllDecorationStringLiterals for getDecorationStringLiteral as well. Same for getAllMemberDecorationStringLiterals as well. It is a nitpick though. Thanks